test/client/ClientServiceTest.java

```diff
@@ -0,0 +1,83 @@
+package client;
+
+import static org.junit.Assert.assertEquals;
+
+import entities.Client;
+import enums.IocServicesEnum;
+import exceptions.services.EntityNotFoundException;
+import exceptions.services.IdNullPointerException;
+import exceptions.services.NullEntityException;
+import ioc.Ioc;
+import java.util.List;
+import java.util.Optional;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.AfterAll;
+import services.implementation.ClientServiceImpl;
+import util.fixtures.ClientServiceFixtures;
+
+public class ClientServiceTest {
+  private ClientServiceImpl clientService;
+
+  @Before
+  public void setup() {
+    clientService = Ioc.getInstance().get(IocServicesEnum.CLIENT_SERVICE_INSTANCE);
+    clientService.insert(ClientServiceFixtures.getClient1());
```

Comment:
Don't do this, I want to know how find by id and  find all behaves with no data.
Setup the in the individual tests